Enigmatry.Entry.Blueprint.Api.Tests/Infrastructure/ApiWebApplicationFactory.cs

```diff
@@ -35,23 +43,33 @@ protected override void ConfigureWebHost(IWebHostBuilder builder)
 
         builder.ConfigureAppConfiguration((_, configBuilder) =>
         {
+            // if the appSettings.json and appSettings.Development.json are messing too much with tests
+            // un-comment out the next line.
+            // configBuilder.Sources.Clear();
```

Comment:
We should clear the configBuilder.Sources, right? I'll check why the tests are failing.

Reply:
They are failing because authentication is enabled in the tests. We need to copy AD settings to the TestConfiguration so that we can clear the sources.